docs/getting-started-guides/vsphere.md

```diff
@@ -40,201 +40,114 @@ guide](/docs/concepts/storage/volumes/#vspherevolume)
+export GOVC_INSECURE=1
 ```
-
-For the VSAN policy based volume provisioning feature, the following additional privileges are required.
-
+* Find Node VM Paths
 ```
-Network > Assign network
-Virtual machine > Configuration > Add new disk
-Virtual Machine > Inventory > Create new
-Virtual machine > Configuration > Add new disk
-Resource > Assign virtual machine to resource pool
+govc ls /datacenter/vm/<vm-folder-name>
 ```
-
-* Provide the cloud config file to each instance of kubelet, apiserver and controller manager via ```--cloud-config=<path to file>``` flag. Cloud config [template can be found at Kubernetes-Anywhere](https://git.k8s.io/kubernetes-anywhere/phase1/vsphere/vsphere.conf)
-
-Sample Config:
-
+* Set disk.EnableUUID to true for all VMs
 ```
-[Global]
-        user = <User name for vCenter>
-        password = <Password for vCenter>
-        server = <IP/URL for vCenter>
-        port = <Default 443 for vCenter>
-        insecure-flag = <set to 1 if the host above uses a self-signed cert>
-        datacenter = <Datacenter to be used>
-        datastore = <Datastore to use for provisioning volumes using storage classes/dynamic provisioning>
-        working-dir = <Folder in which VMs are provisioned, can be null>
-        vm-uuid = <VM Instance UUID of virtual machine which can be retrieved from instanceUuid property in VmConfigInfo, or also set as vc.uuid in VMX file. If empty, will be retrieved from sysfs (requires root)>
-[Disk]
-    scsicontrollertype = pvscsi
+govc vm.change -e="disk.enableUUID=1" -vm='VM Path'
 ```
+Note: If Kubernetes Node VMs are created from template VM then `disk.EnableUUID=1` can be set on the template VM. VMs cloned from this template, will automatically inherit this property.
 
-* Set the cloud provider via ```--cloud-provider=vsphere``` flag for each instance of kubelet, apiserver and controller manager.
-
-* When upgrading to 1.6 install the default storage class addons, [click here for more details](https://github.com/kubernetes/kubernetes/issues/40070)
-
-#### Known issues
-
-### Kube-up (Deprecated)
 
-Kube-up.sh is no longer supported and is deprecated. The steps for kube-up are included but going forward [kube-anywhere](https://github.com/kubernetes/kubernetes-anywhere) is preferred.
+**[ Step 4 ]** Create and assign Roles to the vSphere Cloud Provider user and vSphere entities.
 
-The recommended version for kube-up is [v1.4.7](https://github.com/kubernetes/kubernetes/releases/tag/v1.4.7)
+vSphere Cloud Provider requires the following minimal set of privileges to interact with vCenter. Please refer [vSphere Documentation Center](http://pubs.vmware.com/vsphere-65/index.jsp?topic=%2Fcom.vmware.vsphere.security.doc%2FGUID-18071E9A-EED1-4968-8D51-E0B4F526FDA3.html&resultof=%22%43%72%65%61%74%65%22%20%22%63%72%65%61%74%22%20%22%43%75%73%74%6f%6d%22%20%22%63%75%73%74%6f%6d%22%20%22%52%6f%6c%65%22%20%22%72%6f%6c%65%22%20) to know about steps for creating a Custom Role, User and Role Assignment.
```

Comment:
This part is kind of complicated...
Is there a way to create a super user directly to include all the privileges, or it has to be done like the the following, to assign all those roles/privileges? Just curious...